database/tables/variants.pg

```diff
@@ -2,6 +2,7 @@ columns
     authn_user_id: bigint
     broken: boolean default false
```

Comment:
I assume we'll remove `broken` in a followup PR?

Reply:
Correct, yeah. This PR is the one that stops reading from and writing to this column.